pkg/northbound/gnmi/set.go

```diff
@@ -208,10 +221,41 @@ func (s *Server) buildUpdateResults(targetUpdates mapTargetUpdates,
 		}
 		networkChanges[store.ConfigName(configName)] = changeID
 	}
-
 	return updateResults, networkChanges, nil
 }
 
+func listenForDeviceResponse(changes mapNetworkChanges, target string, name store.ConfigName) (bool, error) {
```

Comment:
I think you could reply only with the err - bool not needed

Reply:
done